README.md

```diff
@@ -0,0 +1,14 @@
+<p align="center">
```

Comment:
FWIW, users are unlikely to ever read any of this, so I'd say it can be removed altogether.

Reply:
The whole README? I'd prefer to keep it for anyone who accidentally stumbles across the repo or address any Flatpak specific issues the application might have (like with custom game install locations as described below).

Reply:
Oh I noticed I still had a download link in the header of readme that points to the installer of the Windows version.

I cleaned it up to only include the header image (as I initially intended) in 97d52283364de6e790a1461960bfcfdce669aee0